Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -166,6 +167,7 @@ FrameSelection::FrameSelection(Document* document)
     , m_caretPaint(true)
     , m_isCaretBlinkingSuspended(false)
     , m_focused(document && document->frame() && document->page() && document->page()->focusController().focusedFrame() == document->frame())
+    , m_isActive(m_document && m_document->page() && m_document->page()->focusController().isActive())
```

Comment:
Should we make a function that’s shared for this expression? I know this is not a lot of code, but I think it would be nice to have a named function to emphasize that it’s the same all three times.

Reply:
Added `isPageActive`.